apps/gro/fields/address-outside.js

```diff
@@ -1,19 +1,8 @@
 'use strict';
 
 module.exports = {
-  'address-text-one': {
+  'address-text': {
     validate: ['required'],
-  },
-  'address-text-two': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-three': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-four': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-five': {
-    labelClassName: 'visuallyhidden'
+    'ignore-defaults': true
```

Comment:
@gxxm this setting will remove all default formatters - we should still use the others `trim` and `hyphens` - pass `formatter: ['trim', 'hyphens']`